sale_merge_draft_invoice/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
```

Comment:
Some files of this module are AGPL and others LGPL. Please, maintain full consistency to one option.

Reply:
:target: https://www.gnu.org/licenses/lgpl

Reply:
added!